crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -299,156 +527,370 @@ impl BuildSpecCommand {
 			"Need help? Learn more at {}\n",
 			style("https://learn.onpop.io").magenta().underlined()
 		))?;
-
 		Ok("spec")
 	}
+
+	// Customize a chain specification.
+	fn customize(&self) -> anyhow::Result<()> {
```

Comment:
Just fix the 2 warnings here and we are good to merge

Reply:
Thank you, resolved